code/modules/antagonists/revolution/revolution.dm

```diff
@@ -258,25 +255,6 @@
 		C.Unconscious(100)
 	owner.remove_antag_datum(type)
 
-/datum/antagonist/rev/proc/check_for_announcement()
-
-	if(rev_team.announced_revolution)
-		return
-
-	var/alive = 0
-	var/revolutionaries = 0
-
-	for(var/mob/player in GLOB.player_list)
-		if(player.stat != DEAD)
-			if(is_revolutionary(player) || is_head_revolutionary(player))
-				++revolutionaries
-			else
-				++alive
```

Comment:
Wouldn't the problem be that it's revs OR alive? Like If there were 5 revs and 15 non revs, that'd be 5/15 (0.33%) instead of 5/20 (0.25%), and not an accurate check of how much of the crew has been converted?

Reply:
The main issue is that no matter how good the code is, it's impossible to test due to needing either a huge amount of alternative accounts OR live testing.
I've decided to go with a more simple option in general simply to make sure it will work as intended.